css/index.css

```diff
@@ -143,4 +143,158 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* sprint challenge
+
+html {
+    font-size:62.5%;
+}
+*/
+
+
+a{
+    text-decoration: none;
+    color: black;
+}
+
+/* Desktops and laptops ----------- */
+@media only screen  and (min-width : 1000px) and (max-width : 2000px) {
+    /* Styles */
+  
+
```

Comment:
Watch the extra spaces here. Try to be consistent throughout the entire CSS file.